translations/monero-core_es.ts

```diff
@@ -1854,226 +1864,235 @@ Please upgrade or connect to another daemon</source>
     </message>
     <message>
         <location filename="../main.qml" line="407"/>
         <source>Waiting for daemon to stop...</source>
-        <translation type="unfinished"></translation>
+        <translation>Esperando a que el daemon se detenga...</translation>
     </message>
     <message>
         <location filename="../main.qml" line="484"/>
         <source>Can&apos;t create transaction: Wrong daemon version: </source>
-        <translation type="unfinished"></translation>
+        <translation>No se pudo crear la transacción: Versión de daemon incorrecta:</translation>
     </message>
     <message>
         <location filename="../main.qml" line="486"/>
         <location filename="../main.qml" line="599"/>
         <source>Can&apos;t create transaction: </source>
-        <translation type="unfinished"></translation>
+        <translation>No se puede crear la transacción:</translation>
     </message>
     <message>
         <location filename="../main.qml" line="494"/>
         <location filename="../main.qml" line="495"/>
         <location filename="../main.qml" line="607"/>
         <location filename="../main.qml" line="608"/>
         <source>No unmixable outputs to sweep</source>
-        <translation type="unfinished"></translation>
+        <translation>No hay salidas no-mezclables que barrer</translation>
     </message>
     <message>
         <location filename="../main.qml" line="507"/>
         <location filename="../main.qml" line="620"/>
         <source>Confirmation</source>
-        <translation type="unfinished"></translation>
+        <translation>Confirmación</translation>
     </message>
     <message>
         <location filename="../main.qml" line="508"/>
         <location filename="../main.qml" line="621"/>
         <source>Please confirm transaction:
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Por favor confirme la transacción:
+</translation>
     </message>
     <message>
         <location filename="../main.qml" line="509"/>
         <source>
 Address: </source>
-        <translation type="unfinished"></translation>
+        <translation>Dirección:</translation>
     </message>
     <message>
         <location filename="../main.qml" line="510"/>
         <source>
 Payment ID: </source>
-        <translation type="unfinished"></translation>
+        <translation>ID de pago:</translation>
     </message>
     <message>
         <location filename="../main.qml" line="511"/>
         <location filename="../main.qml" line="622"/>
         <source>
 
 Amount: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+
+Cantidad:</translation>
     </message>
     <message>
         <location filename="../main.qml" line="512"/>
         <location filename="../main.qml" line="623"/>
         <source>
 Fee: </source>
-        <translation type="unfinished"></translation>
+        <translation>
```

Comment:
These 2 lines should be replaced by:  `<translation type="unfinished"></translation>` and the previous one by 
Comisión: `</source>`